moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -377,7 +377,33 @@ void MotionPlanningFrame::updateQueryStateHelper(moveit::core::RobotState& state
     state.setToDefaultValues(jmg, v);
 }
 
-void MotionPlanningFrame::populatePlannersList(const moveit_msgs::PlannerInterfaceDescription& desc)
+void MotionPlanningFrame::populatePlannersList(const std::vector<moveit_msgs::PlannerInterfaceDescription>& desc)
+{
+  ui_->planning_pipeline_combo_box->clear();
+
+  planner_descriptions_ = desc;
+  size_t default_planner_index = 0;
+  for (auto& d : planner_descriptions_)
+  {
+    // Use planner as default if no pipeline id is given
+    if (d.pipeline_id.empty())
```

Comment:
I guess that's the only location, where you use the info that the default pipeline has no description.
Why not accessing the `default_planning_pipeline` parameter to identify it?